kolibri/deployment/default/settings/dev.py

```diff
@@ -11,3 +11,5 @@
 REDIRECT_WEBPACK = True
 
 INTERNAL_IPS = ['127.0.0.1']
+
+ROOT_URLCONF = 'kolibri.deployment.default.dev_urls'
```

Comment:
This is a great addition, it nicely separates out the two layouts and forces us to consider different url configurations.